stix_shifter_modules/aws_athena/stix_translation/json/ocsf_from_stix_map.json

```diff
@@ -77,6 +162,58 @@
       ]
     }
   },
+  "process": {
+    "fields": {
+      "binary_ref.name": [
+        "file.name"
+      ],
+      "command_line": [
+        "process.cmd_line"
+      ],
+      "created": [
+        "process.created_time"
+      ],
+      "extensions.'x-ocsf-process-ext'.integrity": [
+        "process.integrity"
+      ],
+      "extensions.'x-ocsf-process-ext'.integrity_id": [
+        "process.integrity_id"
+      ],
+      "extensions.'x-ocsf-process-ext'.is_system": [
+        "process.is_system"
+      ],
+      "extensions.'x-ocsf-process-ext'.lineage": [
+        "process.lineage"
+      ],
+      "extensions.'x-ocsf-process-ext'.loaded_modules": [
+        "process.sandbox"
+      ],
+      "extensions.'x-ocsf-process-ext'.terminated_time": [
+        "terminated_time"
+      ],
+      "extensions.'x-ocsf-process-ext'.tid": [
+        "process.tid"
+      ],
+      "x_unique_id": [
+        "process.uid"
+      ],
+      "extensions.'x-ocsf-process-ext'.xattributes": [
+        "process.xattributes"
+      ],
+      "mime_type": [
+        "mime_type"
+      ],
+      "modified": [
```

Comment:
I can't find process.modified in the STIX standard for either 2.0 or 2.1

Reply:
corrected